### PR TITLE
stopping ios change text after reset

### PR DESCRIPTION
**Pull request by armandgray**

### CODE REVIEW

File: src/MentionsTextInput.js [hunk 1]
```diff
@@ -206,7 +211,7 @@ export default class MentionsTextInput extends Component {
   }
 
   isSelectionReplaced() {
-    return this.triggerMatrix
```
Comment (greghe) [inline src/MentionsTextInput.js:209]: good catch, I never would have seen that extra space

File: src/MentionsTextInput.js [hunk 2]
```diff
@@ -76,11 +76,16 @@ export default class MentionsTextInput extends Component {
   }
 
   resetTextbox() {
+    this.isResetting = true;
```
Comment (greghe) [inline src/MentionsTextInput.js:86]: When adding new member variables, it's a good practice to declare them above the constructor:

`isResetting: boolean;`

It's also a good practice to add declarations for existing members, time permitting.

**Comment by greghe** [comment]
Please add `isRetting` declaration.

File: src/MentionsTextInput.js [hunk 3]
```diff
@@ -12,6 +12,13 @@ import PropTypes from 'prop-types';
 const SET_STATE_DELAY = 50;
 
 export default class MentionsTextInput extends Component {
+  lastTextLength: number;
+  lastTriggerIndex: number;
+  triggerMatrix: Array<Object>;
+  isResetting: boolean;
+  isTrackingStarted: boolean;
+  isSelectionChangeHandled: boolean;
```
Comment (greghe) [inline src/MentionsTextInput.js:20]: You win today's "go the extra mile award"

**Comment by greghe** [comment]
👍 👍

**Review by gabrielfirmacion — APPROVED** [review]
🥇

**Pull request merged**